### DISCUSSION TITLE
Navigation buttons doesn't work

### DISCUSSION THREAD

**Discussion by aitormendez** [nolimits4web/swiper#5839] [Q&A] [answered]
Hi, I need help with this basic functionality. I have the standard nav buttons:

```
<div class="swiper-button-next"></div>
<div class="swiper-button-prev"></div>
```

And the most simple JS possible ([here in context](https://github.com/aitormendez/angelgiraldez/blob/main/site/web/app/themes/sage/resources/scripts/TimelineAbout.js#L22-L29)):

```
this.slidesV = document.querySelectorAll('.swiper');

for (const slider of this.slidesV) {
  new Swiper(slider, {
    // allowTouchMove: false,
    navigation: true,
  });
}
```

[An this is the slider online](https://angelgiraldez.e451.net/en/about/). Next and prev buttons are rendered, but it doesn't work when are clicked. Slides works because it can be moved by dragging it. Am I missing something? Thanks!

**Comment by Arhell** [comment]
Try add 
```
navigation: {
          nextEl: ".swiper-button-next",
          prevEl: ".swiper-button-prev",
 },
```
example https://codesandbox.io/s/k86ez1?file=/index.html:2280-2386

**Reply by aitormendez** [reply]
Same result. I think they are two equivalent options.

**Comment by aitormendez** [comment, marked as answer]
I was missing 

`Swiper.use([Navigation]);`

https://github.com/nolimits4web/swiper/issues/3716#issuecomment-662849957

**Reply by neptunewong1998** [reply]
thank you for your reply. I am facing the same issue.

i followed the [getting start pages](https://swiperjs.com/get-started#download-assets)  in swipper.
My swipper is located inside a grid-container.

i added the following code. however, the navigation seems doesn't work. can you tell me what I am missing??
```
    <script>
        Swiper.use([Navigation]);

        var swiper = new Swiper('.swiper', {
            navigation: {
                nextEl: '.swiper-button-next',
                prevEl: '.swiper-button-prev',
            },
        });

    </script>
```
```
    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/npm/swiper/swiper-bundle.min.css"
    />

<script src="https://cdn.jsdelivr.net/npm/swiper/swiper-bundle.min.js"></script>
```
```  
                <div class="swiper">
                  <div class="swiper-wrapper">
                    <div class="swiper-slide">Slide 1</div>
                    <div class="swiper-slide">Slide 2</div>
                    <div class="swiper-slide">Slide 3</div>
                    <div class="swiper-slide">Slide 4</div>
                  </div>
                  <div class="swiper-button-next"></div>
                  <div class="swiper-button-prev"></div>
                </div>
```
